examples/gno.land/p/demo/forms/create.gno

```diff
@@ -0,0 +1,95 @@
+package forms
+
+import (
+	"std"
+	"time"
+
+	"gno.land/p/demo/json"
+	"gno.land/p/demo/ufmt"
+)
+
+func CreateField(label string, fieldType string, required bool) Field {
+	return Field{
+		Label:     label,
+		FieldType: fieldType,
+		Required:  required,
+	}
+}
+
+func ParseDates(openAt string, closeAt string) (*time.Time, *time.Time) {
+	var openAtTime, closeAtTime *time.Time
+
+	dateFormat := "2006-01-02T15:04:05Z"
+
+	// Parse openAt if it's not empty
+	if openAt != "" {
+		res, err := time.Parse(dateFormat, openAt)
+		if err != nil {
+			panic(ufmt.Errorf("invalid date: %v", openAt))
+		}
+		openAtTime = &res
+	}
+
+	// Parse closeAt if it's not empty
+	if closeAt != "" {
+		res, err := time.Parse(dateFormat, closeAt)
+		if err != nil {
+			panic(ufmt.Errorf("invalid date: %v", closeAt))
+		}
+		closeAtTime = &res
+	}
+
+	return openAtTime, closeAtTime
+}
+
+func (db *FormDatabase) CreateForm(title string, description string, openAt string, closeAt string, data string) string {
+	// Parsing dates
+	openAtTime, closeAtTime := ParseDates(openAt, closeAt)
+
+	// Parsing the json submission
+	node, err := json.Unmarshal([]byte(data))
+	if err != nil {
+		ufmt.Errorf("invalid json: %v", err)
+	}
+
+	fieldsCount := node.Size()
+	fields := make([]Field, fieldsCount)
+
+	// Parsing the json submission to create the gno data structures
+	for i := 0; i < fieldsCount; i++ {
+		field, err := node.GetIndex(i)
+		if err != nil {
+			ufmt.Errorf("error: %v", err)
+		}
+
+		labelNode, _ := field.GetKey("label")
```

Comment:
You should probably handle these errors

Reply:
handled in https://github.com/gnolang/gno/pull/2604/commits/f0a0e585f1bbba47cbff41c010c9e5deaa0e4f5c